docker-hub/oci-artifacts.md

```diff
@@ -0,0 +1,175 @@
+---
+title: OCI artifacts on Docker Hub
+keywords: oci, artifacts, docker hub
+description: >
+  You can store any software artifacts on Docker Hub, not just container images
+---
+
+You can use Docker Hub to store any kind of software artifact, not just
+container images. Docker Hub supports OCI artifacts, as defined in the
+[OCI artifacts specification](https://github.com/opencontainers/image-spec/blob/main/artifact.md).
```

Comment:
This is wrong. This is exactly the contentious point of the recent OCI Artifact kerfuffle (https://github.com/opencontainers/image-spec/pull/999).

**TL;DR**: "artifact" is an overloaded term; OCI decided to define a dedicated manifest for it (described in the `artifact.md` spec), but Jon raised some concerns with it and wants to drop the artifact manifest before it's accepted to the official release; the discussion continues about whether "artifacts" as (regular manifest + config manfiest -- this is what Docker Hub supports) should continue to be the only way to represent artifacts OR if we should accept the new artifact manifests. In other words: Docker Hub supports the [image] manifest+config but not the standalone not yet released `artifact` manifest